support/loader/loader.js.ejs

```diff
@@ -7,6 +7,15 @@
       var args = Array.prototype.slice.call(arguments);
       args.unshift(method);
       <%= locals.globalNamespace %>.push(args);
+      // if we have methods expecting a callback
+      // and the lib already failed to load
+      // don't block the loop
+      var lastIndex = args.length - 1;
+      var callback = typeof args[lastIndex] === 'function' && args[lastIndex];
+      var failed = <%= locals.globalNamespace %>.__state === 'failed';
+      if (failed && callback) {
+        callback();
```

Comment:
Shouldn't  the library that is expecting the callback be responsible for ignoring it; instead of us forcing it? what about something like `callback(new Error('Failed to load metrics'))`? then the one expecting the callback could choose to ignore it but we still have made clear that there was an error.